src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshalling/PointerArrayMarshaller.cs

```diff
@@ -59,7 +59,11 @@ public static ReadOnlySpan<IntPtr> GetManagedValuesSource(T*[]? managed)
         /// <param name="numElements">The unmanaged element count.</param>
         /// <returns>The <see cref="Span{TUnmanagedElement}"/> of unmanaged elements.</returns>
         public static Span<TUnmanagedElement> GetUnmanagedValuesDestination(TUnmanagedElement* unmanaged, int numElements)
-            => new Span<TUnmanagedElement>(unmanaged, numElements);
+        {
+            if (unmanaged is null)
+                return [];
```

Comment:
```suggestion
                return [];

```

Reply:
Just looking consistency. If you want to make it a single like, you could also just do this:

```csharp
=> unmanaged is null ? [] : new Span<TUnmanagedElement>(unmanaged, numElements);
```

or

```csharp
=> => new Span<TUnmanagedElement>(unmanaged, (unmanaged is null ? 0 : numElements));
```